pwndbg/dbg/gdb.py

```diff
@@ -111,6 +136,25 @@ def bottom_frame(self) -> pwndbg.dbg_mod.Frame:
         return GDBFrame(value)
 
 
+class GDBMemoryMap(pwndbg.dbg_mod.MemoryMap):
+    def __init__(self, reliable_perms: bool, qemu: bool, pages: Sequence[pwndbg.lib.memory.Page]):
```

Comment:
Isn't it better to make the `reliable_perms` a 3rd argument with a default of `True` or `False`?

Reply:
Maybe? Though, we always set it, anyway, so I don't see how much of a benefit that would bring?